_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -66,6 +66,7 @@ $brand-secondary-accent:    rgb(242, 108, 170) !default;
 $success: $green !default;
 $warning: $yellow !default;
 $error: $red !default;
+$shadow: rgba(0, 0, 0, .25);
```

Comment:
This new variable should use either 1) a value from our grayscale-cool color palette or 2) extend (i.e. ``transparentize()``) a value from our grayscale-cool color palette.